web/src/components/tasks/LicenseTask.test.tsx

```diff
@@ -215,11 +218,11 @@ describe("<LicenseTask />", () => {
 
     it("does not display error alert when license status is found", async () => {
       renderTask();
-
       mockApi.checkLicenseStatus.mockResolvedValue(generateUserData({}));
       fireEvent.submit(screen.getByTestId("check-status-submit"));
+
       await waitFor(() => {
-        expect(screen.getByText("Pending")).toBeInTheDocument();
+        expect(screen.getByText(Config.licenseSearchTask.completedStatusText)).toBeInTheDocument();
```

Comment:
There are a few places where `pending` is changind to `completed`. They should all be looked at